ticket.php

```diff
@@ -302,19 +281,16 @@
     "unmerge"                   => "Unmerge",
     "close_and_lock"            => "Kapat ve Kilitle",
 
-    "block_warning"             => "Kullanıcının e-postası da engellenir ve artık talep açamaz.",
-
     "mass_reply"                => "Toplu Yanıt",
 
     "due_today"                 => "Bugün süresi doluyor",
     "overdue"                   => "Süresi Geçmiş",
     "unassigned"                => "Atanmayanlar",
 
     "pause_duetime_desc"        => "Bu talebin etkin bir SLA planı varsa, takip tarihinden sonra kalan kalan süreyi duraklatın. Teslim süresi ancak talebe bir cevap veya not eklendikten sonra tekrar başlayacaktır (takip dahil).",
-    "delete_follow_up"          => "İzlemeyi Sil",
 
     "add_cc"                    => "CC Ekle",
-//    "reply_above_line"          => "Lütfen bu satırın üstünde cevap verin",
+    "reply_above_line"          => "Please reply above this line",
```

Comment:
This line should probably be uncommented and kept as it was?

Reply:
Changed it to that text.